server/mongoose-handlers/languages.js

```diff
@@ -0,0 +1,29 @@
+const mongoose = require("mongoose");
+
+// Not sure how to implement just yet
+const languagesModel = require("../models/languages");
+
+module.exports = {
+  createLanguages: async () => {
+    const createLanguages = new languagesModel({
+      languages: [
+        "Python",
+        "JavaScript",
+        "Java",
+        "PHP",
+        "GoLang",
+        "C#",
+        "C++",
+        "Ruby",
+      ],
+    });
```

Comment:
This should be done with a seed file and script, not recreated each time the server boots.

Reply:
Seed created, run with nodemon seeder/seedLanguages.js